interp/constrintern.mli

```diff
@@ -45,13 +45,15 @@ type var_internalization_type =
 
 type var_internalization_data =
     var_internalization_type *
-      (** type of the "free" variable, for coqdoc, e.g. while typing the
-	  constructor of JMeq, "JMeq" behaves as a variable of type Inductive *)
+      (* type of the "free" variable, for coqdoc, e.g. while typing the
```

Comment:
Is there no way to put a comment on part of a tuple type?

Reply:
No that I know of, other than using a record. Likely there are some mantis / gihub issues about this. Recall that `ocamldoc` has seen little to 0 development over the years.